processing/src/main/java/io/druid/query/aggregation/BufferAggregator.java

```diff
@@ -178,4 +178,24 @@ default void relocate(int oldPosition, int newPosition, ByteBuffer oldBuffer, By
   {
   }
 
+  /**
+   * returns true if aggregator's output type is primitive long/double/float and aggregated value is null,
+   * but when aggregated output type is Object, this method always returns false,
+   * and users are advised to check nullability for the object returned by {@link #get()}
+   * method.
+   * The default implementation always return false to enable smooth backward compatibility,
+   * re-implement if your aggregator is nullable.
+   *
+   * @param buf      byte buffer storing the byte array representation of the aggregate
+   * @param position offset within the byte buffer at which the aggregate value is stored
+   *
+   * @return true if the aggrgeated value is null otherwise false.
+   * The default implementation always return false to enable smooth backward compatibility,
```

Comment:
I think don't need to repeat this once again.

Reply:
done.